curriculum/challenges/english/14-responsive-web-design-22/learn-css-flexbox-by-building-a-photo-gallery/6153a3952facd25a83fe8083.md

```diff
@@ -7,65 +7,50 @@ dashedName: step-20
 
 # --description--
 
-Create a media query for screens smaller than `800px` in width. In that media query, create a `#gallery img` rule and set the `width` property to `50%`. This will convert your gallery to a two-column layout.
+Great job! You have created a responsive image gallery using flexbox. You can test the responsive layout by changing the width of the preview window. The images start in a three-column layout and stack as needed depending on the width of the window.
```

Comment:
These instructions are quite lengthy, any chance you can try to shorted it a little @lasjorg? Also, there isn't a test for this step. Wasn't there a way to test this? I don't think we should have a step without a test. I don't personally mind it in some cases, but I don't think it's the way we want to go.

Reply:
I definitely think we need a test here. As far as I know, we don't have any other steps in RWD that don't have a test - maybe somewhere in backend or something.

Reply:
I updated this step.

I removed all the congratulations and "try testing the layout" text. I also added the tests for the CSS so it is now a requirement.